Automattic/sensei
# Fix contact teacher full width in learning mode

Resolves the issue of the Contact teacher button becoming full width in learning mode in Divi and Astra

## Proposed Changes
* Fixed contact teacher button becoming full width in learning mode. It was probably a side effect of [this change](https://github.com/Automattic/sensei/commit/d813122c15410a7169ce9e616331fe25d84e16e7).

## Testing Instructions


1. Add a course with a lesson
2. Add a contact teacher button in the lesson
3. Make sure learning mode is on
4. Enable Astra theme
5. Access the lesson as student from frontend
6. Make sure contact teacher button isn't full width.

#### Before
![Screenshot 2023-11-14 at 2 36 19 PM](https://github.com/Automattic/sensei/assets/6820724/ad9c0016-4baa-4b66-8d46-ccda5902d951)

##### After
![Screenshot 2023-11-14 at 2 36 33 PM](https://github.com/Automattic/sensei/assets/6820724/48666ccc-f9e9-409f-9893-dcc50f28f7b9)

## Pre-Merge Checklist
